pages/common/zed.md

```diff
@@ -0,0 +1,24 @@
+# zed
+
+> Text editor designed to be fast, efficient and convenient.
+> More information: <https://zed.dev>.
```

Comment:
```suggestion
> More information: <https://zed.dev/docs/#cli>.
```
Consider a better documentation link. I found this on the site but it's extremely barebones.